api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -45,6 +45,25 @@ message TlsParameters {
     TLSv1_3 = 4;
   }
 
+  enum CompliancePolicy {
+    NONE = 0;
+
+    // FIPS_202205 configures a TLS connection to use:
```

Comment:
I wonder where does FIPS_202205 comes from?
I assume there's some standard on the web that describes it. If there is, please add a link.
(my goal here is 1) let the users know where it came from, 2) if a new version is added and one wants to support it - they'll know where to look for the spec).

Reply:
This is lifted straight from boringssl code: https://boringssl.googlesource.com/boringssl/+/refs/heads/main/include/openssl/ssl.h#5921. I suspect it's due to the validation certifications (https://boringssl.googlesource.com/boringssl/+show/master/crypto/fipsmodule/FIPS.md#17). Putting a version seems meaningful as both FIPS and boringssl can evolve - it is plausible CHACHA20 might become NIST approved, and there'd be no need to restrict TLSv1.3, for example.